### PR TITLE
Unit test, njk tweaks

### PR DESCRIPTION
**Pull request by lynnette-orr**

### CODE REVIEW

File: server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.test.ts [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+])
+
+njkEnv.addFilter('formatDate', formatDate)
+njkEnv.addFilter('formatIsAccredited', formatIsAccreditedFilter)
+njkEnv.addFilter('formatFunctionalSkillType', formatFunctionalSkillTypeFilter)
+
+const prisonerSummary = aValidPrisonerSummary()
+const template = 'overviewTabContentsV2.njk'
+
+describe('overviewTabContents', () => {
```
Comment (nathanrussell-moj-digital) [inline server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.test.ts:26]: Excellent unit tests 👍 Nice work 👍

File: assets/scss/local.scss [hunk 2]
```diff
@@ -21,6 +21,28 @@
...
 }
 
+// Custom column with vertical border
+
+.app-table-column-with-right-border {
+  border-right: 1px solid $govuk-border-colour;
+  display: inline-block;
+}
+
+.govuk-grid {
```
Comment (nathanrussell-moj-digital) [inline assets/scss/local.scss:31]: You and your css wizardry 🤣 🧙‍♀️

File: server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk [hunk 3]
```diff
@@ -20,32 +18,33 @@
...
-                    <span>3</span> <span class="govuk-tag govuk-tag--grey">Archived</span><br><br>
-                    <p class="govuk-body govuk-!-margin-bottom-0">View archived goals</p>
-                  </td>
-                </tr>
-              </tbody>
-            </table>
-            <p class="govuk-hint govuk-!-font-size-16 govuk-!-margin-top-0" data-qa="goal-last-updated-hint">
-              {{ 'params.lastUpdatedLabel' | default('Last updated on') }} {{ 'params.goal.updatedAt' | formatDate('DD MMMM YYYY') }}<span class="govuk-!-display-none-print"> by {{ 'params.goal.updatedByDisplayName' }}{{ ', ' + 'params.goal.updatedAtPrisonName' if 'params.goal.updatedAtPrisonName' }}</span>
-            </p>
+            <div class="govuk-grid">
```
Comment (nathanrussell-moj-digital) [inline server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk:26]: Not that I mind, but why did you change it from a govuk styled table to divs using grid css layout?

I guess semantically its more correct? It was never really tabular data, we were perhaps using the table for layout!!! ? 😬 😱 

So maybe this is more semantically correct?

Like I say, I dont mind, just intrigued as to the reason 😁

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Comment by nathanrussell-moj-digital** [comment]
Nice work , and approved 👍 ✅ 

My only suggestion moving forwards is that for PRs like this, where the thing that is being changed/delivered is a screen of some sort, it would be useful to include a screenshot of the resultant screen in the PR description. That then gives reviewers a little more context of the change etc 👍

File: server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk (hunk 3 above)
Comment (lynnette-orr) [reply server/views/pages/overview/partials/overviewTab/overviewTabContentsV2.njk:26]: There are vertical dividing lines which I added in after this initial code. It was tricky getting them to line up properly so I ended up having to use a lot of padding, margins etc. It was displaying weirdly in the Cypress ui and I was worried in case maybe smaller screens would have difficulty with it - probably not an issue but the grid layout seemed to solve the Cypress display weirdness.
